packages/dids/src/methods/did-dht.ts

```diff
@@ -1175,13 +1179,13 @@ export class DidDhtDocument {
       const publicKeyBase64Url = Convert.uint8Array(publicKeyBytes).toBase64Url();
 
       // Define the data for the DNS TXT record.
-      const txtData = [`id=${methodId}`, `t=${keyType}`, `k=${publicKeyBase64Url}`];
+      const txtData = [`t=${keyType}`, `k=${publicKeyBase64Url}`];
```

Comment:
so id is out now because of updated spec?

Reply:
Yes, see https://did-dht.com/#verification-methods.